lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
Although if we update the documentation for dropdown, and explain the sub-components  bit more, it might clarify their usage and we could go with `dropdown-button`